lib/api/api.go

```diff
@@ -703,6 +704,12 @@ func (s *service) postDBRevert(w http.ResponseWriter, r *http.Request) {
 	go s.model.Revert(folder)
 }
 
+func (s *service) postDBCleanEnc(w http.ResponseWriter, r *http.Request) {
+	var qs = r.URL.Query()
+	var folder = qs.Get("folder")
+	go s.model.CleanEnc(folder)
```

Comment:
Why not just revert?
I'd argue that revert/override are already the same thing, (effectively it's still a revert just in a different location), so adding a new endpoint seems like a nono, you can branch what revert does based on the folder type.

Reply:
There's already override and revert, doing similar but clearly different things. This is again something "similar", but again different from both. The consistent continuation is adding a new endpoint. So you want to deprecate the override endpoint and switch to revert only? I am fine with that. I'd like to wait for @calmh's ack or not before going ahead (just because that's going to be a bunch of boring busywork that I don't want to just revert again).

Reply:
I think we can keep Override (make remotes like local) and Revert (make local like remote). This, then, is a variant of Revert.